MAUI/Calendar/selections.md

```diff
@@ -0,0 +1,249 @@
+
+>**NOTE** You can select the cells in the Year, Decade, and Century views in the calendar only when the `AllowViewNavigation` property is set to false.
+
+## Single selection
+The `Single` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Single`. In this selection, you can select a single date at a time from the calendar.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Single">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Single;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view single selection in .NET MAUI Calendar.](images/selection/maui-month-view-single-selection.png)
+{% endcolumn %}
+
+>**NOTE**
+* In the Year, Decade, and Century views, you can select the cells, only when the `AllowViewNavigation` property is set to false.
+* In this scenario, the `selection changed` event will return the first date of the month, year, decade or century of the selected cell.
+Eg: 
+* In the year view, when the Dec month cell is selected then the selected date value will be 01-12-2022.
+* In the decade view, when the (2022) year cell is selected then the selected date value will be 01-01-2022.
+* In the century view, when the (2020-2029) decade cell is selected then the selected date value will be 01-01-2020.
+
+
+## Multiple selection
+The `Multiple` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Multiple`. In this selection, you can select multiple dates from the cell. If you want to remove the selected cell, click the same cell again.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Multiple">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Multiple;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Multiple selection in .NET MAUI Calendar.](images/selection/maui-multiple-selection.png)
+
+
+## Range selection
+The `Range` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Range`. In this Selection, you can able to select a range of dates from the cell by interacting the cell either tap or swipe action. Swipe action can only performed by setting the [EnableSwipeSelection](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_EnableSwipeSelection) property to `true` in the calendar.
```

Comment:
select a range of dates from the cell by interacting the cell -- Replace cell with calendar

select a range of dates from the calendar by interacting the cell

Reply:
Replaced the cell by calendar